tinygrad/tensor.py

```diff
@@ -306,7 +306,9 @@ def min(self, axis=None, keepdim=False): return -((-self).max(axis=axis, keepdim
   def mean(self, axis=None, keepdim=False):
     out = self.sum(axis=axis, keepdim=keepdim)
     return out * (prod(out.shape)/prod(self.shape))
-
+  def std(self, axis=None, keepdim=False, unbiased=True):
+    square_sum = ((self - self.mean(axis=axis, keepdim=True)).square()).sum(axis=axis, keepdim=keepdim)
+    return (square_sum / ((prod(self.shape[:axis+1]) if axis is not None else prod(self.shape))-(1 if unbiased else 0))).sqrt()
```

Comment:
prod(square_sum.shape)!

Reply:
Right, it does not pass. I see, I will try something else.

Reply:
My last commit `prod(square_sum[:axis])` passes tests, does it handle higher axes?